.circleci/config.yml

```diff
@@ -203,6 +211,14 @@ jobs:
           name: docker-driver
           command: make docker-driver-push
 
+  publish/binaries:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: github release
+          command: make publish
```

Comment:
I think we need `BUILD_IN_CONTAINER=false` here because we call clean as a pre-req and some of the loki/promtail pre-reqs build yacc and proto